apps/src/code-studio/initApp/project.js

```diff
@@ -1914,7 +1900,7 @@ var projects = (module.exports = {
         .then(({data, jqXHR}) => {
           currentSourceVersionId =
             jqXHR && jqXHR.getResponseHeader('S3-Version-Id');
-          unpackSources(data);
+          unpackSources(data, true);
```

Comment:
is the `true` parameter needed here?

Reply:
Nice catch of a relic from my first approach. Thanks!